File: simple-erp-gui/src/app/models/trade/SalesAndExpensesData.ts
import {ChartData} from "./ChartData";

export interface SalesAndExpensesData{
  quantity: ChartData[];
  sale: ChartData[];
  purchase: ChartData[];
  quantityToday: string;
  quantityMonth: string;
  saleToday: string;
  saleMonth: string;
  purchaseToday: string;
  purchaseMonth: string;
}

File: simple-erp-app/src/main/java/com/simpleerp/simpleerpapp/dtos/trade/SalesAndExpensesData.java
package com.simpleerp.simpleerpapp.dtos.trade;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.ArrayList;
import java.util.List;

@Getter
@Setter
@NoArgsConstructor
public class SalesAndExpensesData {
    private List<ChartData> quantity = new ArrayList<>();
    private List<ChartData> sale = new ArrayList<>();
    private List<ChartData> purchase = new ArrayList<>();
    private String quantityToday;
    private String quantityMonth;
    private String saleToday;
    private String saleMonth;
    private String purchaseToday;
    private String purchaseMonth;
}
